tests/jballerina-unit-test/src/test/resources/test-src/object/object_langlib_function_invocation_negative.bal

```diff
@@ -58,3 +58,9 @@ public class DynamicListenerImpl {
 public readonly class StackFrameImpl {
     *runtime:StackFrame;
 }
+
+function testAnonObjectImpl() {
```

Comment:
Is this correct file for this test? There's no langlib function call, right?

Reply:
Removed the test as the test scenario is already covered.